src/core/environment.go

```diff
@@ -516,22 +516,82 @@ func processors() (res []*proto.CpuInfo) {
 }
 
 func processorCache(item cpu.InfoStat) map[string]string {
+	cache := getProcessorCacheInfo(cpuid.CPU)
+	if cpuid.CPU.Supports(cpuid.SSE, cpuid.SSE2) {
+		cache["SIMD 2:"] = "Streaming SIMD 2 Extensions"
+	}
+	return cache
+}
+
+func getProcessorCacheInfo(cpuInfo cpuid.CPUInfo) map[string]string {
+	cache := getDefaultProcessorCacheInfo(cpuInfo)
+
+	out, err := exec.Command("lscpu").Output()
+	if err != nil {
+		log.Warnf("Error executing lscpu on host: %v", err)
+		return cache
+	}
+
+	return parselscpuInfo(string(out), cache)
+}
+
+func parselscpuInfo(lscpuInfo string, cache map[string]string) map[string]string {
```

Comment:
nit: naming: From https://go.dev/doc/effective_go#mixed-caps

> the convention in Go is to use MixedCaps or mixedCaps rather than underscores to write multiword names.

maybe:

- `parseLscpuInfo`
- `parseLscpu` (the "info" suffix doesn't add much meaning), 
- `parseLscpuOutput`
- `addLscpuCache` - using "add" as our verb to indicate we're adding the `lscpu` cache data to the provided cache data?

Reply:
Fixed.